__tests__/topics.test.js

```diff
@@ -118,3 +119,55 @@ describe('CORE: GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('CORE: GET /api/articles', () => {
+  test('200: should return an array of objects', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(typeof articles[0]).toBe('object');
+      });
+  });
+  test('200: should return an array with the length 5', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toHaveLength(5);
+      });
+  });
+  test('200: the returned objects should have the correct properties', () => {
+    const exampleObject = {
+      comment_count: '2',
+      author: 'icellusedkars',
+      title: 'Eight pug gifs that remind me of mitch',
+      article_id: 3,
+      topic: 'mitch',
+      created_at: '2020-11-03T09:12:00.000Z',
+      votes: 0,
```

Comment:
I would probably do this more dynamically like article_id :expect.any(Number) within your match object to make it dynamic :)